tests/e2e/scenarios/test_journey_coverage.py

```
@@ -0,0 +1,247 @@
+            "    pass\n"
+        ),
+        (
+            "@pytest.mark.xfail(run=False, reason='disabled')\n"
+            "def test_required_journey():\n"
+            "    pass\n"
+        ),
+    ],
+)
+def test_python_evidence_rejects_disabled_tests(source: str):
+    """A named Python test cannot satisfy the gate while disabled."""
+    with pytest.raises(AssertionError, match="disabled by .* marks"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)